### PR TITLE
Refactor Tests to use Test Helpers

### PR DESCRIPTION
**Pull request by southpolesteve**
Only one test failing! Thanks typescript. This started out as trying to convert all the tests to use `getTestDatabase` but ended up doing many things:

- Remove all `// tslint:disable:max-line-length` comments. Prettier handles this automatically.
- Convert test config file to export { endpoint, masterKey } 
- Allow `getTestDatabase` and `removeAllDatabases` to use default client
- Swap argument order of `getTestDatabse` and `getTestContainer` so `client` is easier to omit
- Remove lots of unnecessary comments, client instantiations, and comments

### CODE REVIEW

File: src/test/integration/aggregateQuery.spec.ts [hunk 1]
```diff
@@ -5,14 +5,11 @@ import { Container, ContainerDefinition, Database } from "../../client";
...
-import testConfig from "./../common/_testConfig";
+import { endpoint, masterKey } from "./../common/_testConfig";
 import { TestData } from "./../common/TestData";
 import { bulkInsertItems, getTestContainer, removeAllDatabases } from "./../common/TestHelpers";
 
 // process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
-
-const endpoint = testConfig.host;
-const masterKey = testConfig.masterKey;
 const client = new CosmosClient({ endpoint, auth: { masterKey } });
```
Comment (christopheranderson) [inline src/test/integration/aggregateQuery.spec.ts]: Probably don't need a client here, either, right?

File: src/test/integration/documentClient.spec.ts [hunk 2]
```diff
@@ -1,14 +1,12 @@
 ﻿import * as assert from "assert";
 import { Constants, CosmosClient, IHeaders } from "../../";
-import testConfig from "./../common/_testConfig";
+import { endpoint, masterKey } from "./../common/_testConfig";
+
+const client = new CosmosClient({ endpoint, auth: { masterKey } });
```
Comment (christopheranderson) [inline src/test/integration/documentClient.spec.ts]: Remove unnecessary client

Comment (southpolesteve) [reply src/test/integration/documentClient.spec.ts]: This one is still used here. But this test is also testing `client.documentClient` directly so we should probably just delete it entirely...

**Comment by christopheranderson** [comment]
VSTS failed. Kicked off another build.

**Comment by christopheranderson** [comment]
Build 12 succeeded, but VSTS doesn't seem to track that the status has changed: https://cosmos-db-sdk-public.visualstudio.com/cosmos-db-sdk-public/_build/results?buildId=12&view=results 

I'm treating it as a passing on both travis and VSTS, and merging.

File: src/test/integration/documentClient.spec.ts (hunk 2 above)
Comment (christopheranderson) [reply src/test/integration/documentClient.spec.ts]: Ah yeah, documentClient leaking is still a hack we've done to expose the session management stuff until we refactor documentClient away.

**Pull request merged**